lib/phoenix/live_dashboard/page_builder.ex

```diff
@@ -666,11 +685,74 @@ defmodule Phoenix.LiveDashboard.PageBuilder do
     {LayeredGraphComponent, assigns}
   end
 
+  ## Helpers
+
+  @doc """
+  Encodes Sockets for URLs.
```

Comment:
For the encode functions, what if we add small examples like:

    <button phx-click="show-info" phx-value-info=<%= encode_socket(@socket) %>

?